src/main/kotlin/no/nav/familie/ef/sak/beregning/OmregningService.kt

```diff
@@ -87,8 +86,8 @@ class OmregningService(private val behandlingService: BehandlingService,
         val innvilgelseOvergangsstønad =
                 vedtakHistorikkService.hentVedtakForOvergangsstønadFraDato(fagsakId,
                                                                            YearMonth.from(nyesteGrunnbeløpGyldigFraOgMed))
-        if (innvilgelseOvergangsstønad.samordningsfradragType != null && samordningsfradrag == null) {
-            logger.info(MarkerFactory.getMarker("G-Omberegning"),
+        if (innvilgelseOvergangsstønad.inntekter.all { (it.samordningsfradrag ?: BigDecimal.ZERO) > BigDecimal.ZERO }) {
```

Comment:
Burde det være any? Hvis noen har inntekt fra mai som har samordningsfradrag > 0? 
```suggestion
        if (innvilgelseOvergangsstønad.inntekter.any { (it.samordningsfradrag ?: BigDecimal.ZERO) > BigDecimal.ZERO }) {
```